espnet/nets/pytorch_backend/e2e_asr_mix_transformer.py

```diff
@@ -0,0 +1,452 @@
+#!/usr/bin/env python3
+# encoding: utf-8
+
+# Copyright 2019 Shigeki Karita
```

Comment:
Could you summarize which part did you change from the original `espnet/nets/pytorch_backend/e2e_asr_transformer.py` with your authorship for this file?

Reply:
I will modify this. Thanks!

Reply:
Could you add more information of how it is different from `espnet/nets/pytorch_backend/e2e_asr_transformer.py`?